sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Models/RepositoryProperties.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Containers.ContainerRegistry
+{
+    /// <summary> Repository attributes. </summary>
+    public partial class RepositoryProperties
+    {
+        /// <summary>
+        /// Gets an instance of <see cref="RepositoryProperties"/>.
+        /// </summary>
+        public RepositoryProperties()
+        {
+        }
+
+        /// <summary> Delete enabled. </summary>
+        public bool? CanDelete { get; set; }
+        /// <summary> Write enabled. </summary>
+        public bool? CanWrite { get; set; }
+        /// <summary> List enabled. </summary>
+        public bool? CanList { get; set; }
+        /// <summary> Read enabled. </summary>
+        public bool? CanRead { get; set; }
+        /// <summary> Enables Teleport functionality on new images in the repository improving Container startup performance. </summary>
```

Comment:
nit - the wording sounds a bit like a method.
```suggestion
        /// <summary> Gets or sets whether Teleport functionality is enabled on new images in the repository. Setting this to true can improve Container startup performance. </summary>
```